src/helm/benchmark/run_expander.py

```diff
@@ -323,6 +324,17 @@ class NumTrainTrialsRunExpander(ReplaceValueRunExpander):
     }
 
 
+class MaxEvalInstancesRunExpander(ReplaceValueRunExpander):
+    """For varying number of val/text examples."""
+
+    name = "max_eval_instances"
+    values_dict = {
+        "vhelm_default": [100],
```

Comment:
Rename vhelm to heim?  Is heim going to be the permanent name?

Reply:
Renamed everything to `heim`.